schemas/blocks/list_add_edit_remove_question.json

```diff
@@ -0,0 +1,69 @@
+{
+  "$id": "https://eq.ons.gov.uk/blocks/list_add_edit_remove_question.json",
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "block": {
+    "type": "object",
+    "properties": {
+      "id": {
+        "$ref": "https://eq.ons.gov.uk/common_definitions.json#/identifier"
+      },
+      "number": {
+        "type": "string"
+      },
+      "cancel_text": {
+        "type": "string"
+      },
+      "type": {
+        "type": "string",
+        "enum": [
+          "ListAddQuestion",
+          "ListEditQuestion",
+          "ListRemoveQuestion"
+        ]
+      },
+      "show_on_section_summary": {
+        "type": "boolean"
+      },
+      "question": {
+        "$ref": "https://eq.ons.gov.uk/questions/definitions.json#/question"
+      },
+      "question_variants": {
+        "$ref": "https://eq.ons.gov.uk/blocks/definitions.json#/question_variants"
+      },
+      "list_summary": {
```

Comment:
This should be broken out into a separate definition that can be referenced by this and the original question block json.